app/Http/Controllers/Api/ActivityController.php

```diff
@@ -27,11 +27,22 @@ public function index(Request $request)
     {
         // Create an empty Signup query and eager-load posts, which we
         // can either filter or paginate to retrieve all signup records.
-        $query = $this->newQuery(Signup::class)->with('posts');
+        if ($request->query('orderBy') === 'desc') {
+            $query = $this->newQuery(Signup::class)->with('posts')->orderBy('created_at', 'desc');
+        } else {
+            $query = $this->newQuery(Signup::class)->with('posts');
+        }
```

Comment:
I think you can clean up these conditionals a bit, since you only need to orderby, if the param is passed

```php
$query = $this->newQuery(Signup::class)->with('posts');

if ($request->query('orderBy') === 'desc') {
  $query = $query->orderBy('created_at', 'desc');
}
```

Reply:
oh yeah duh! I don't know why I did it the other way.